lms/djangoapps/teams/views.py

```diff
@@ -525,6 +526,13 @@ def post(self, request):
         if course_key and not has_team_api_access(request.user, course_key):
             return Response(status=status.HTTP_403_FORBIDDEN)
 
+        topic_id = request.data.get('topic_id')
```

Comment:
We should have a view unit test to properly cover this code path, right?

Reply:
Thank you Simon, now the python unit test case is added, and I also added another unit test case for Javascript code. Could you take another look?